src/gov-action-contracts/AIPs/SecurityCouncilMgmt/GovernanceChainSCMgmtActivationAction.sol

```diff
@@ -49,7 +49,7 @@ contract GovernanceChainSCMgmtActivationAction {
     }
 
     function perform() external {
-        IUpgradeExecutor upgradeExecutor = IUpgradeExecutor(l2AddressRegistry.coreGov().owner());
+        UpgradeExecutor upgradeExecutor = UpgradeExecutor(l2AddressRegistry.coreGov().owner());
```

Comment:
why are we not using the interface?

Reply:
because `EXECUTOR_ROLE` isn't part of the interface

Reply:
i guess we could update the interface in the `upgrade-executor` package, not sure what's best

Reply:
https://github.com/OffchainLabs/upgrade-executor/pull/4

Reply:
IAccessControl can be imported separately

Reply:
oh sweet, will update

Reply:
cbc5c7d

Reply:
there seems to be a few other places where IUpgradeExecutor can be used instead of UpgradeExecutor